src/test/regress/expected/multi_multiuser.out

```diff
@@ -523,55 +523,8 @@ $cmd$);
  usage_access
 (2 rows)
 
--- a user with all privileges on a schema should be able to distribute tables
```

Comment:
maybe we can keep `create_distributed_table` test and drop the rest?

Reply:
oops, yes